src/utilities.ts

```diff
@@ -157,3 +158,29 @@ export function getRequestInformation(
         }
     }
 }
+
+export async function encryptKMS(plaintext: string, keyId: string): Promise<string> {
```

Comment:
It'll be more efficient if encrypt/decrypt is incorporated in logger directly.

Reply:
Fields to encrypt can be passed in through the log metadata fields. This way the logger can encrypt fields as instructed through the metadata and whoever viewing the log would also get information on which fields are encrypted.